cs9/Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM @BASE_IMAGE_NAME@
-RUN dnf install -y \
-    automake bash bzip2 bzip2-libs bzip2-devel coreutils-single e2fsprogs e2fsprogs-libs \
-    perl file file-libs fontconfig freetype gcc-c++ git glibc krb5-libs libaio \
-    libcom_err libcom_err-devel libgomp libICE \
+
+ADD bin/rpm-pkg-info.sh /tmp/rpm-pkg-info.sh
+
+RUN dnf install -y bash && ./tmp/rpm-pkg-info.sh BASE &&\
```

Comment:
@aandvalenzuela , why `./tmp/rpm-pkg-info.sh` (relative path) and not `/tmp/rpm-pkg-info.sh` ?

Reply:
I think this is same for `alma8`

Reply:
I thought both were fine since `cwd` is `/`, but let me change that